src/main/java/hackaton/cuckoodobot/CuckoodoBot.java

```diff
@@ -41,6 +41,42 @@
     private String somethingWentWrong = "\uD83D\uDCA9 Что-то пошло не так! \uD83D\uDCA9";
     private long groupId;
 
+    private static String helpMessage = "Привет! Я кукуду-туду-лист с напоминалками, буду куковать тебе о важных вещах, чтобы ты не забыл.\n" +
+            "Используй команду /helpMessage для помощи В)\n" +
+            "Список моих команд:\\n\" +\n" +
+            "   [] -- опционально. По умолчанию -- весь чат.\n" +
+            "/add - добавить задачу\n" +
+            "    /добавить задача [@username]\n" +
+            "/list - вывести все задачи\n" +
+            "    /список\n" +
+            "/done - пометить задачу как готовую\n" +
+            "    /готово 1\n" +
+            "/del - удалить задачу\n" +
+            "    /удалить 1\n" +
+            "/assign - назначить задачу\n" +
+            "    /назначить 1 [@username]\n" +
+            "/helpMessage - вывести эту помощь\n" +
+            "/fullhelp - полный список команд\n" +
+            "/eng - if you speak only english, use this command. But we are recommend use russian text(and fullhelp)!";
+
+    private static String fullHelpMessage = "/add: добавить, задача, напомнить, д\n" +
+            "/list: список, все, всё, в\n" +
+            "/done: готово, готов, сделать, сделано, сделаль, выполнено, разделался, г\n" +
+            "/del: удалить, убрать, у\n" +
+            "/assign: назначить, навесить, перевестистрелки, н\n" +
+            "/helpMessage: помощь, хелп, памагите, ничегонепонимаю, п\n" +
+            "/fullhelp: всяпомощь, ещепомощь, команды, ещекоманды, в\n" +
+            "/eng: english, englishplease";
+
+    private static String enghelpMessage = "add - add a issue\n" +
+            "list - view a list with all issues\n" +
+            "done - mark issue as done\n" +
+            "del - delete issue\n" +
+            "assign - assign issue\n" +
+            "helpMessage - print helpMessage\n" +
+            "fullhelp - print full helpMessage\n" +
+            "eng - print helpMessage in english";
+
     private final Pattern intervalPattern = Pattern.compile("(\\d{1,2} d)?(\\d{1,2} h)?(\\d{1,2} min)?");
```

Comment:
remove that 

`private final Pattern intervalPattern = Pattern.compile("(\\d{1,2} d)?(\\d{1,2} h)?(\\d{1,2} min)?");`